.husky/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+# npx lint-staged
```

Comment:
Why is this commented out?

Reply:
Didn't realise i committed this 🥹. was getting a weird lint error after updating the `next.config.js` file which was blocking my commits. i had to manually comment that line out to enable me commit my changes. reverting now 🏃‍♂️🏃‍♂️